src/ecMatching.ml

```diff
@@ -59,6 +61,7 @@ module Zipper = struct
   | ZWhile  of expr * spath
   | ZIfThen of expr * spath * stmt
   | ZIfElse of expr * stmt  * spath
+  | ZMatch  of expr * spath * int * (EcIdent.t * ty) list * ((EcIdent.t * ty) list * stmt) list
```

Comment:
Use a record (could be a inlined record)

Reply:
Fixed in latest changes.